docs/examples/website-change.md

```diff
@@ -0,0 +1,176 @@
+
+```yaml
+  - id: get-old-headers
+    type: getter
+    transition: check-site
+    variables:
+      - key: lastModified
+        scope: workflow
+      - key: etag
+        scope: workflow
+```
+
+## Compare Values
+Now that we both the current and previous header values we can make a comparison and check whether the website has changed using the switch state `check-site`. 
+
+The switch state below has three possible conditions. The first condition is used for validation and will `transition` to the error state `unsupported-site` if neither `etag` or `lastModified` was fetched from the current headers. The last two are to check if either of the `etag` or `lastModified` values have changed between the previous and current headers. If either one of these headers has changed it means that the website has changed and the property `siteChanged` is set to true. If none of these conditions are satisfied,  the `siteChanged` property is set to false because we can assume that no errors/changes have occurred.
+
+```yaml
+  - id: check-site
+    type: switch
+    defaultTransition: save-values
+    defaultTransform: ". += {siteChanged: false}"
+    conditions:
+      - condition: ".etag == null and .lastModified == null"
+        transition: unsupported-site
+      - condition: ".etag != .var.etag"
+        transition: save-values
+        transform: ". += {siteChanged: true}"
+      - condition: ".lastModified != .var.lastModified"
+        transition: save-values
+        transform: ". += {siteChanged: true}"
+  - id: unsupported-site
+    type: error
+    error: unsupported.site
+    message: "https://docs.direktiv.io is not supported: site must respond with atleast one of these headers: ['Etag', 'Last-Modified']"
+```
+
+## Save current values
+Finally we save the current headers to the `lastModified` and `etag` workflow variables, so next time this workflow is executed they can retrived in the `get-old-headers` state.
+
+```yaml
+  - id: save-values
+    type: setter
+    variables:
+      - key: lastModified
+        scope: workflow
+        value: .lastModified
+      - key: etag
+        scope: workflow
+        value: .etag
+```
+
+## Sample Output
+Note: the getter state will place variables into the `var` property. So the `var.etag` and `var.lastModified` values are the old headers.
+
+```json
+{
+	"etag": "\"60d55d9b-54b1\"",
+	"lastModified": "Fri, 25 Jun 2021 04:37:47 GMT",
+	"siteChanged": false,
+	"var": {
+		"etag": "\"60d55d9b-54b1\"",
+		"lastModified": "Fri, 25 Jun 2021 04:37:47 GMT"
+	}
+}
+```
+
+## Extra - Converting to a Cron Job
+This workflow can currently run as is, and be manually executed. However this example is more than likely to be used as a [cron job](walkthrough/scheduling.html#cron). To convert this workflow all you need to do is add the start block to the top of the workflow. Below is an example that if added to the workflow, will run this workflow every once every two hours.
```

Comment:
> Below is an example that if added to the workflow, will run this workflow every once every two hours.

"Below is an example that, if added to the workflow, will run this workflow every once every two hours."

(added a comma)